com.unity.render-pipelines.high-definition/Documentation~/HDRP-Camera.md

```diff
@@ -59,30 +59,30 @@ The Camera Inspector includes the following groups of properties:
+|| **SMAA Quality Preset**         | Use the drop-down to select the quality of SMAA. The difference in resource intensity is small between **Low** and **High**.<br />&#8226; **Low**: The lowest SMAA quality. This is the least resource-intensive option.<br />&#8226; **Medium**: A good balance between SMAA quality and resource intensity.<br />&#8226; **High**: The highest SMAA quality. This is the most resource-intensive option. This property only appears when you select **Subpixel Morphological Anti-aliasing (SMAA)** from the **Anti-aliasing** drop-down. |
+| |**TAA Quality Preset**         | The quality level of TAA. The default settings for higher presets aren't guaranteed to produce better results than lower presets. The result depends on the content in your scene. However, the high quality presets give you more options that you can use to adapt the anti-aliasing to your content. |
 | |**TAA Sharpen Strength**    | The intensity of the sharpening filter that Unity applies to the result of TAA. This reduces the soft look that TAA can produce. High values can cause ringing issues (dark lines along the edges of geometry) |
 | |**TAA History Sharpening**  | This strength of the history sharpening effect. When this value is above 0, Unity samples the history buffer with a bicubic filter that sharpens the result of TAA. You can use this to produce a sharper image during motion. A high value can cause ringing issues (dark lines along the edges of geometry).If you set this value to 0, it increases the performance of TAA because Unity simplifies the history buffer sampling|
 | |**TAA Anti-flickering**     | The strength of TAA's anti-flickering effect. Use this to reduce some cases of flickering. When you increase this value, the risk o  [ghosting](Glossary.md#Ghosting) or [disocclusion](Glossary.md#Disocclusion) artifacts. <br />This property is only visible when **TAA Quality Preset** is set to a value above **Low**. |
-| |**TAA Speed rejection**       | Controls the threshold at which Unity rejects history buffer contribution for TAA. You can increase this value to remove ghosting artifacts. This works because Unity rejects history buffer contribution when a GameObject's current speed and reprojected speed history are very different. When you increase this value, it might also reintroduce some aliasing for fast-moving GameObjects. Controls the threshold at which Unity rejects history buffer contribution for TAA. Increasing this value can help to remove ghosting artifacts because Unity rejects history buffer contribution when a GameObject's current speed and reprojected speed history are very different. While this can be effective in reducing ghosting, it might also reintroduce some aliasing for fast-moving GameObject. When you set this value to 0, it increases the performance of TAA because Unity does not process speed rejection. |
```

Comment:
I think we are loosing information here by removing this statement. Maybe work with @FrancescoC-unity about a wording more appropriate for the doc?